rules/linux/execution_apt_binary.toml

```
@@ -24,10 +24,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m 
-  [process where process.name : ("apt", "apt-get") and process.args : "changelog"] 
-  [process where process.name : "sensible-pager" and process.args : ("/bin/sh", "/bin/bash") and 
-   process.parent.name : ("apt", "apt-get")]
+process where event.type == "start" and process.parent.name in ("apt", "apt-get") and process.parent.args == "changelog" and process.name == "sensible-pager" and process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") and process.parent.name in ("apt", "apt-get")
```

Keep the format. Also, easier if we group the process fields and parent process fields together (I like to start with all process info first, usually leading with the name)

```suggestion
process where event.type == "start" and process.name == "sensible-pager" and 
  process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") and 
  process.parent.name in ("apt", "apt-get") and process.parent.args == "changelog"
```